src/pages/lock-exception/AddLockExceptionForm.js

```diff
@@ -93,22 +92,7 @@ class AddLockExceptionForm extends Component {
                     })
                 })
                 .catch(error => {
-                    if (this.isPageMounted()) {
-                        const messageError =
-                            error && error.message
-                                ? error.message
-                                : i18n.t(i18nKeys.messages.unexpectedError)
-
-                        this.context.updateAppState({
-                            showSnackbar: true,
-                            loading: false,
-                            snackbarConf: {
-                                type: ERROR,
-                                message: messageError,
-                            },
-                            pageState: { ...this.state },
-                        })
-                    }
+                    console.error(error)
```

Comment:
Why did you remove the info from the snackbar?

Reply:
I had to remove the snackbar component due to dependency issues and so this code no longer had an effect.